src/KlaviyoAPI.php

```diff
@@ -314,14 +314,17 @@ private function trimPath ( $path )
     }
 
     /**
-     * Return decoded json response as associative array.
+     * Return decoded json response as associative or empty array.
      *
      * @param string $response
      * @return mixed
      */
     private function decodeJsonResponse( $response )
     {
-        return json_decode( $response, true );
+        if (!empty($response)) {
+            return json_decode( $response, true );
+        }
+        return json_decode( '{}', true );
```

Comment:
mind just adding a comment as to why we would return this? Ex. 
```suggestion
        // Delete requests return empty responses 
        return json_decode( '{}', true );
```

Reply:
Let's add this in the docstring instead then we can add the additional detail Kamil provided in the pr comments as well so it's crystal clear e.g. `Return decoded json response as associative or empty array. Certain Klaviyo endpoints return an empty string on success and PHP >=7 throws a syntax error when trying to decode it.`

Reply:
That should be now implemented :)